src/Ucommerce.Sitecore.Cli/Commands/Install.cs

```diff
@@ -38,13 +41,29 @@ public virtual async ValueTask ExecuteAsync(IConsole console)
 
             var baseDirectory = new DirectoryInfo(AppContext.BaseDirectory);
             var sitecoreDirectory = new DirectoryInfo(SitecorePath);
-            var connectionStringLocater = new SitecoreInstallationConnectionStringLocator(ConnectionString);
+            var connectionStringLocator = new SitecoreInstallationConnectionStringLocator(ConnectionString);
             var sitecoreDbAvailabilityService = new SitecoreDatabaseAvailabilityService();
-            var runtimeVersionChecker = new RuntimeVersionChecker(connectionStringLocater, logging);
-            var updateService = new UpdateService(connectionStringLocater, runtimeVersionChecker, sitecoreDbAvailabilityService);
+            var runtimeVersionChecker = new RuntimeVersionChecker(connectionStringLocator, logging);
+            var updateService = new UpdateService(connectionStringLocator, runtimeVersionChecker, sitecoreDbAvailabilityService);
             var versionChecker = new SitecoreVersionCheckerOffline(sitecoreDirectory, logging);
-            var installStep = new InstallStep(baseDirectory, sitecoreDirectory, versionChecker, connectionStringLocater, updateService, runtimeVersionChecker, logging);
 
+            if (!NoDatabase)
```

Comment:
We should consider renaming this, a double negative in a check is not very clear

Reply:
I thought about it as well, do you have any ideas considering the property's set to true when a the flag is given - maybe another ordering in the logic?